webapp/locales/en.json

```diff
@@ -25,7 +25,16 @@
     "shouted": "Shouted",
     "commented": "Commented",
     "userAnonym": "Anonymous",
-    "socialMedia": "Where else can I find"
+    "socialMedia": "Where else can I find",
+    "network": {
+      "title": "Network",
+      "following": "is following:",
+      "followingNobody": "follows nobody.",
+      "followedBy": "is followed by:",
+      "followedByNobody": "is not followed by anyone.",
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-07T22:07:26Z" title="Saturday, June 8th 2019, 12:07:26 am +02:00">Jun 8, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -25,7 +25,16 @@
     "shouted": "Shouted",
     "commented": "Commented",
     "userAnonym": "Anonymous",
-    "socialMedia": "Where else can I find"
+    "socialMedia": "Where else can I find",
+    "network": {
+      "title": "Network",
+      "following": "is following:",
+      "followingNobody": "follows nobody.",
+      "followedBy": "is followed by:",
+      "followedByNobody": "is followed by nobody.",
```


this would be better `is not followed by anyone`

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-11T21:26:20Z" title="Tuesday, June 11th 2019, 11:26:20 pm +02:00">Jun 11, 2019</time>_

---


Thanks for the commit!